### DISCUSSION TITLE
Help need on front end

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#3647] [Ideas]
Hi,
   We are building a CMS using grapesJs. We have created componets for header, footer,products slider, different kind of banners. We have completed almost a Ecommerce site using grapesjS + React Js +NodeJs and Mongdb. We are able to store and load components to and from DB. 

our web admin side is for example abc.com/editor 

Here we are creating layout and storing and loading pages

we are facing issue on front end. for example, abc.com/home

Here we want to display stored components and want to bind dynamic data . We dont wat to use editor on front end. Please suggest how is it possible. As this is not mentioned in the Grapes JS docs.

**Comment by codingmachine16** [comment]
Hi @artf  Please guide us

**Reply by artf** [reply]
> Here we want to display stored components and want to bind dynamic data . We dont wat to use editor on front end

What do you mean by that? Try to explain better, please

**Reply by codingmachine16** [reply]
For example , we add a link component <a href="abc" >test text</a>

This is stored in DB. we want to display it on a webpage in a way that test text should be dynamic data and is coming from a DB or json file. how to bind dynamic data with component and replace text test text with dynamic data

**Reply by artf** [reply]
Read more about [here](https://grapesjs.com/docs/modules/Components-js.html).

**Reply by codingmachine16** [reply]
Issue resolved. Thanks alot

**Reply by anshmarelab85** [reply]
hi @codingmachine16  how can i see your web builder product
how can i contact with you.
can you please provide me your email?
